### DISCUSSION TITLE
Setting default browser / How to use "am start" with URL?

### DISCUSSION THREAD

**Discussion by uwe-007** [termux/termux-packages#11642] [Q&A] [answered]
Please allow one question regarding termux not directly.

Each time after rebooting my Fire TV stick (Android 9 / FireOS) and opening an URL with "am start ..." I get asked with which app I want to open it.

1. Can I set the default browser somewhere at boot time / in .bashrc?
2. How can I send an URL with "am start" to firefox?
This doesn't work: am start -n org.mozilla.firefox/.App -d  "https://github.com"

**Comment by uwe-007** [comment, marked as answer]
Ok, found answer to question 2:
am start -a android.intent.action.VIEW -d "https://github.com" org.mozilla.firefox

**Comment by ZaxonXP** [comment]
Is there a way to set up default browser from termux?
